### PR TITLE
Upgrade to Django 1.8.14

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@safwanrahma r?

**Comment by safwanrahman** [comment]
Looks good. Test passes :+1: @glogiotatidis 
According to our convention, we do not include the wheel of hashes. Can you please remove that?
Except that **r+wc**

**Comment by glogiotatidis** [comment]
> According to our convention, we do not include the wheel of hashes. Can you please remove that?

Can you please explain why we don't include hashes of wheels?

**Comment by safwanrahman** [comment]
@glogiotatidis The reason of not using wheel is written [here ](https://github.com/mozilla/kitsune/blob/master/peep.sh#L14)

We always pass ``--no-use-wheel`` while downloading dependencies. So thats the reason we should not include the hash of wheel.

**Comment by jayy7312** [comment]
> 

- Right

**Review by dow572 — COMMENTED** [review]
Flare

File: requirements/default.txt [hunk 1]
```diff
@@ -92,8 +92,8 @@ cryptography==1.4
 # sha256: _eJnrrA6jUnjNB5_caIoxnSNzkO1wA9lWBeA2Ztwstk
 dennis==0.7
 
-# sha256: 7BSL5zVI2gkN12wujFfJjosehPLLh1ALm-VCAYekNfs
-Django==1.8.11
+# sha256: UoLEi5D7spUHKZ9ZIhU3joSaKAj0heQURibmZxXVRk0
```
Comment (jmayes41) [inline requirements/default.txt:95]: *
```suggestion
# sha256: UoLEi5D7spUHKZ9ZIhU3joSaKAj0heQURibmZxXVRk0
```
**

**Review by tinandanny — APPROVED** [review]

**Review by candaceblewett42 — COMMENTED** [review]
image

**Comment by Mikevin** [comment]
@candaceblewett42 Is this someone's personal data? Looks like a privacy concern.

@glogiotatidis Can you check if this is relevant or remove it? Reporting to GitHub itself looks like a slow process.

**Review by Chanvannak52 — COMMENTED** [review]
requirements/default.txt

**Review by candaceblewett42 — COMMENTED** [review]
.

**Review by candaceblewett42 — COMMENTED** [review]
Hey partner

**Comment by tran09** [comment]
Nice work man

**Review by tran09 — COMMENTED** [review]
Nice work

**Review by tran09 — COMMENTED** [review]
What doesn't this do?

**Pull request merged**